.github/workflows/release-testing.yml

```diff
@@ -24,200 +24,241 @@ env:
-          BAZEL_COMMAND: "test"
-          BAZEL_TARGETS: "//rs/tests/..."
-          BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
-          BAZEL_EXTRA_ARGS_RULES: "--test_tag_filters=system_test_hotfix"
-          HONEYCOMB_API_TOKEN: ${{ secrets.HONEYCOMB_API_TOKEN }}
-      - name: Upload bazel-bep
-        uses: actions/upload-artifact@v4
-        if: always()
-        with:
-          name: ${{ github.job }}-bep
-          retention-days: 14
-          if-no-files-found: ignore
-          compression-level: 9
-          path: |
-            bazel-bep.pb
-            profile.json
-  dependency-scan-release-cut:
-    name: Dependency Scan for Release
-    runs-on:
-      group: zh1
-      labels: dind-large
-    container:
-      image: ghcr.io/dfinity/ic-build@sha256:748d5cd92d982531a25cd6c4f247bb6c0e484c9ac654164341f59a7d57d1fffc
-      options: >-
-        -e NODE_NAME --privileged --cgroupns host -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp
-    timeout-minutes: 60
-    env:
-      SHELL_WRAPPER: "/usr/bin/time"
-      CI_PROJECT_PATH: ${{ github.repository }}
-      CI_PIPELINE_ID: ${{ github.run_id }}
-      CI_COMMIT_SHA: ${{ github.sha }}
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
-      SLACK_PSEC_BOT_OAUTH_TOKEN: ${{ secrets.SLACK_PSEC_BOT_OAUTH_TOKEN }}
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          ref: ${{ github.event.workflow_run.head_branch }}
-      - name: Before script
-        id: before-script
-        shell: bash
-        run: |
-          [ -n "${NODE_NAME:-}" ] && echo "Node: $NODE_NAME"
-      - name: Login to Dockerhub
-        shell: bash
-        run: ./ci/scripts/docker-login.sh
-        env:
-          DOCKER_HUB_USER: ${{ vars.DOCKER_HUB_USER }}
-          DOCKER_HUB_PASSWORD_RO: ${{ secrets.DOCKER_HUB_PASSWORD_RO }}
-      - name: Setup python deps
-        id: setup-python-deps
-        shell: bash
-        run: |
-          pip3 install --ignore-installed -r requirements.txt
-      - name: Dependency Scan for Release
-        id: dependency-scan-release-cut
+      - name: Run qualification for version ${{ matrix.version }} from the tip of the branch
         shell: bash
         run: |
-          set -euo pipefail
-          export PYTHONPATH=$PWD/ci/src:$PWD/ci/src/dependencies
-          cd ci/src/dependencies/
-          $SHELL_WRAPPER python3 job/bazel_rust_ic_scanner_release_job.py
-  trigger-qualifier-workflow:
-    name: Qualify
-    needs: [ci-main, bazel-system-test-nightly, bazel-system-test-staging, bazel-system-test-hotfix, dependency-scan-release-cut]
-    uses: dfinity/dre/.github/workflows/qualify.yaml@main
-    secrets: inherit
-    with:
-      version: ${{ github.sha }}
+          bazel test //rs/tests/dre:guest_os_qualification --config=systest --cache_test_results=no --test_timeout=7200 --test_env=OLD_VERSION=${{ matrix.version }}
```

Comment:
I think this should use our baze-test-all action [like we do here for example](https://sourcegraph.com/github.com/dfinity/ic/-/blob/.github/workflows/ci-main.yml?L73-L86) such that bazel is correctly setup.\

Reply:
Will take a look into it, Its a `manual` test and I would love just that test to be run